.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
```

Comment:
`.editorconfig` should be excluded via `.gitattributes`

Reply:
Done